invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -94,20 +94,32 @@ def get_affiliation(self, obj):
                 }
                 identifiers = affiliation.get("identifiers")
                 if identifiers:
-                    # PIDS-FIXME: DataCite accepts only one, how to decide
-                    identifier = identifiers[0]
+                    # FIXME: Make configurable
+                    AFFILIATION_IDENTIFIER_TYPES_PREFERENCE = ("ror", "isni", "gnd")
```

Comment:
naming 😅: would we use this variable somewhere else? It has no relation to "datacite" nor "serialization", but it indicates the preference in that specific case.